Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY: build
+# target: build - Build package, run tests and create distribution
+build: .asdf
+	@mvn
+
+# target: tools - Bootstrap all build tools
+PHONY: tools
+tools: .asdf
+	@touch .envrc
+
+.PHONY: clouseau1
+# target: clouseau1 - Start local inistance of clouseau1 node
+clouseau1: .asdf
+	@mvn scala:run -Dlauncher=$@
```

Comment:
The Erlang cookie is now optional for Clouseau.  It is the launcher interface that has changed.  It should say something like: `mvn scala:run -Dname=clouseau1`.

Reply:
However, I think it could be beneficial to let the user to set the cookie still if that is needed (for example, CouchDB uses its own cookie and the default from `~/.erlang.cookie`).  This can be done by using the `with-cookie` launcher.

```shell
mvn scala:run -Dlauncher=with-cookie  -Dname=clouseau1 -Dcookie=brumbrum
```